assignment-client/src/AssignmentClientApp.cpp

```diff
@@ -44,10 +44,10 @@ AssignmentClientApp::AssignmentClientApp(int argc, char* argv[]) :
     ShutdownEventListener::getInstance();
 #   endif
 
-    setOrganizationName("High Fidelity");
+    setOrganizationName(BuildInfo::MODIFIED_ORGANIZATION);
     setOrganizationDomain("highfidelity.io");
     setApplicationName("assignment-client");
-    setApplicationName(BUILD_VERSION);
+//    setApplicationName(BuildInfo::VERSION);
```

Comment:
should kill this and actually set the version now that we're using our own paths